test/integration/base.py

```diff
@@ -97,25 +195,44 @@ def run_dbt(self, args=None):
         if args is None:
             args = ["run"]
 
+        args = ["--strict"] + args
+
         return dbt.handle(args)
 
     def run_sql_file(self, path):
         with open(path, 'r') as f:
-            return self.run_sql(f.read())
+            statements = f.read().split(";")
+            for statement in statements:
+                self.run_sql(statement)
+
+    # horrible hack to support snowflake for right now
+    def transform_sql(self, query):
+        to_return = query
```

Comment:
Oh, I understand. I was thinking about using an ORM for these seed files. That would make it substantially easier to apply existing integration tests to new adapters. SQLAlchemy has dialect support for postgres/redshift/snowflake, for instance. This works for now though